### DISCUSSION TITLE
how to create route with query params and get query params

### DISCUSSION THREAD

**Discussion by khushmohammad** [tezjs/tezjs#112] [Q&A]
how can i create  route with  query  and get query params

**Comment by ajayojha** [comment]
Please refer to this documentation https://github.com/tezjs/tezjs/blob/main/docs/pages/directory-structure/pages.md

**Reply by khushmohammad** [reply]
thanks for this sir,
but I wish to create url based on query params
http://localhost:3000/product-list/item-name?mobile=mi

how can i get mobile value(mi) from url

**Reply by ajayojha** [reply]
try this `$tezRoutes.queryParams["mobile"]`.

**Reply by khushmohammad** [reply]
it is not working.
$tezRoutes undefined showing
> try this `$tezRoutes.queryParams["mobile"]`.

**Reply by ajayojha** [reply]
Sorry, I missed something. It's not available in the framework, but will be available in the next release :).

**Reply by khushmohammad** [reply]
ok sir
